extensions-core/parquet-extensions/src/test/java/org/apache/druid/data/input/parquet/BaseParquetReaderTest.java

```diff
@@ -39,6 +39,7 @@
 class BaseParquetReaderTest extends InitializedNullHandlingTest
 {
   ObjectWriter DEFAULT_JSON_WRITER = new ObjectMapper().writerWithDefaultPrettyPrinter();
+  protected final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Nit: you can use the object mapper created on line 42 for line 41 and remove the inline instantiation of the object mapper.